tests/query.rs

```diff
@@ -48,8 +48,8 @@ macro_rules! test_cases {
                 tremor_runtime::functions::load()?;
                 let query_dir = concat!("tests/queries/", stringify!($file), "/").to_string();
                 let query_file = concat!("tests/queries/", stringify!($file), "/query.trickle");
-                let in_file = concat!("tests/queries/", stringify!($file), "/in.xz");
-                let out_file = concat!("tests/queries/", stringify!($file), "/out.xz");
+                let in_file = concat!("tests/queries/", stringify!($file), "/in");
```

Comment:
this is just for coverage of the `load_events_file` function, so this case is actually used. It will always support both the `.xz` name and the one without and always try both.